lib/wallet.js

```diff
@@ -1,4 +1,7 @@
 module.exports = function(options) {
+  var EC = require('elliptic').ec;
+  var ec = new EC('secp256k1');
```

Comment:
For `getAddressFromValidationPublicKey`, I needed to get a `sjcl.ecc.point` from just the x coordinate. I couldn't find a way to do that with sjcl, whereas elliptic has `ec.curve.pointFromX`